planning/obstacle_velocity_limiter/README.md

```diff
@@ -0,0 +1,209 @@
+# Obstacle Velocity Limiter
+
+## Purpose
+
+|                                     Without this node | With this node                                      |
+| ----------------------------------------------------: | :-------------------------------------------------- |
+| ![obstacle_velocity_limiter_off](./media/ovl_off.png) | ![obstacle_velocity_limiter_on](./media/ovl_on.png) |
+
+This node limits the velocity when driving in the direction of an obstacle.
+For example, it allows to reduce the velocity when driving close to a guard rail in a curve.
```

Comment:
I prefer this text to come before the picture. Nice picture btw, thanks!

Reply:
Done: https://github.com/autowarefoundation/autoware.universe/pull/1579/commits/1019c4b833a64b7e064ead2a5ffc445be281fe9a